src/mon/Monitor.cc

```diff
@@ -31,6 +31,7 @@
 #include "common/blkdev.h"
 #include "common/cmdparse.h"
 #include "common/signal.h"
+#include "common/MemoryModel.h"
```

Comment:
Names of most files in `common` are written with with lower cases (and underscores).

Reply:
Hi @rzarzynski,

Noted. The reference to MemoryModel is within the existing file structure.I followed the pattern used in [MDCache.cc#L48](https://github.com/ceph/ceph/blob/main/src/mds/MDCache.cc#L48).